dashboard/test/ui/features/acquisition_products/pl_landing_page.feature

```diff
@@ -62,14 +62,13 @@ Feature: Professional Learning landing page
     # Sees Instructor Professional Learning Sections section
     And I wait until element "button:contains(Create a section)" is visible
 
-  @skip
   Scenario: Regional Partner sees relevant content sections
     Given I am a program manager with started and completed courses
     And I am on "http://studio.code.org/my-professional-learning"
```

Comment:
I know this is merged, but in the videos we were seeing a sad bee on this page. Do you have any theories as to why

Reply:
I wasn't able to replicate the sad bee locally 😕 My best guess is that some wires got crossed on the test server since it wasn't getting an exact string match for that button.

Reply:
okay, let's keep an eye on it. My worry is that it's a test server specific issue, which is why you couldn't repro locally, but I don't have any more theories than that...